ui/admin/app/templates/scopes/scope/sessions/index.hbs

```diff
@@ -17,134 +17,143 @@
-              />
-            </B.Td>
-            <B.Td>
-              <Hds::Text::Code>{{B.data.user.id}}</Hds::Text::Code>
-              {{#if B.data.user.accountName}}
-                <br />
-                <Hds::Badge @text={{B.data.user.accountName}} />
-              {{/if}}
-            </B.Td>
-            <B.Td>{{B.data.target.displayName}}</B.Td>
-            <B.Td>
-              <time datetime={{format-date-iso B.data.session.created_time}}>
-                {{format-date-iso-human B.data.session.created_time}}
-              </time>
-            </B.Td>
-            <B.Td>
-              <SessionStatus @model={{B.data.session}} />
-            </B.Td>
-            <B.Td>
-              {{#if (can 'cancel session' B.data.session)}}
-                <Hds::Button
+      {{#if @model.sessions}}
+        <Hds::Table
+          @columns={{array
+            (hash label=(t 'form.id.label'))
+            (hash label=(t 'resources.user.title'))
+            (hash label=(t 'resources.target.title'))
+            (hash label=(t 'form.started.label'))
+            (hash key='status' label=(t 'form.status.label') isSortable=true)
+            (hash label=(t 'titles.actions'))
+          }}
+          @sortBy='status'
+          @model={{@model.sessions}}
+          @density='short'
+          @valign='middle'
+        >
+          <:body as |B|>
+            <B.Tr>
+              <B.Td>
+                <Hds::Copy::Snippet
+                  data-test-session={{B.data.id}}
+                  @textToCopy={{B.data.id}}
                   @color='secondary'
-                  @text={{t 'actions.cancel'}}
-                  {{on 'click' (route-action 'cancelSession' B.data.session)}}
                 />
-              {{/if}}
-            </B.Td>
-          </B.Tr>
-        </:body>
-      </Hds::Table>
-
-    {{else if
-      (has-resource-filter-selections 'scopes.scope.sessions' 'status')
-    }}
-      <Rose::Layout::Centered>
-        <Rose::Message @title={{t 'titles.no-results'}} as |message|>
-          <message.description>
-            {{t 'descriptions.no-results'}}
-          </message.description>
-        </Rose::Message>
-      </Rose::Layout::Centered>
+              </B.Td>
+              <B.Td>
+                {{B.data.user.accountName}}
+              </B.Td>
+              <B.Td>
+                {{B.data.target.displayName}}
+              </B.Td>
+              <B.Td>
+                <time datetime={{format-date-iso B.data.created_time}}>
+                  {{format-date-iso-human B.data.created_time}}
+                </time>
+              </B.Td>
+              <B.Td>
+                <SessionStatus @model={{B.data}} />
+              </B.Td>
+              <B.Td>
+                {{#if (can 'cancel session' B.data)}}
+                  <Hds::Button
+                    @color='secondary'
+                    @text={{t 'actions.cancel'}}
+                    {{on 'click' (route-action 'cancelSession' B.data)}}
```

Comment:
Are we still using `route-action`s?

Reply:
I was trying to keep the scope of this PR limited to just the pagination and filtering but I can update this now if you'd prefer.

Reply:
`cancelSession` is an easy one so I went ahead and moved it from the router to the controller.